test-suite/Makefile

```diff
@@ -250,11 +250,11 @@ $(addsuffix .log,$(wildcard bugs/closed/*.v)): %.v.log: %.v
 # Unit tests
 #######################################################################
 
-OCAMLOPT := $(OCAMLFIND) opt $(CAMLFLAGS)
-SYSMOD:=-package num,str,unix,dynlink,threads
+ifeq ($(LOCAL),true)
+  export OCAMLPATH := $(patsubst "%",%,$(COQLIBINSTALL))
```

Comment:
Please add a comment about what this patsubst is doing.

Reply:
Is this really the correct thing to do?

Reply:
I don't understand it either, please explain what it does.